### ISSUE TITLE
Cannot remove and add same container name/version inproc (in the same JVM)

### ISSUE THREAD

**Issue by ligadata-william** [LigaData/Kamanja#1133]
We are currently unable to remove and re-add a container of the same version inproc because the associates types are not being removed along with the container. This was found from a failed unit test. Note the "Result Description" below. It indicates that higher version of mapofenvcodes exists in the system as opposed to the container itself.
```
[info] - Container Tests *** FAILED ***
[info]   "{
[info]     "APIResults" : {
[info]       "Status Code" : -1,
[info]       "Function Name" : "AddContainerDef",
[info]       "Result Data" : null,
[info]       "Result Description" : "Error :com.ligadata.Exceptions.AlreadyExistsException: Higher active version of Map com.ligadata.kamanja.samples.containers.mapofenvcodes already exists in the systemFailed to Add Container:com.ligadata.kamanja.samples.containers.envcodes.000000000001000000"
[info]     }
[info]   }" did not include substring that matched regex "Status Code" : 0 (ContainerAPISpec.scala:337)
```

**Comment by ligadata-ramana** [comment]
Create a pull request #1178

**Comment by ligadata-william** [comment]
I've verified this is fixed and in 1.5.0. Closing.
